src/routes/index.js

```diff
@@ -23,7 +23,7 @@ const Routes = () => (
       <Route path="/signup" exact component={signupView} />
       <ProtectedRoute path="/profile" exact component={Profile} />
       <ProtectedRoute path="/editprofile" exact component={EditProfile} />
-      <ProtectedRoute path="/article/:slug" exact component={ArticleDetailView} />
+      <Route path="/article/:slug" exact component={ArticleDetailView} />
```

Comment:
why have you removed the protected route component

Reply:
We decided as a team that one should be able to read an article/view even when not logged in but can not make any changes to it until they log in